doofinder-for-woocommerce/includes/api/class-update-on-save-api.php

```diff
@@ -91,12 +91,14 @@ private function sendRequest($url, $data, $method = 'POST')
 
         $response = wp_remote_request($url, $data);
 
-        if (!is_wp_error($response) && $response['response']['code'] === 200) {
```

Comment:
I think it would be better to change the order, because in this case if `$response` is a `WP_Error` it will give an error when accessing the response field as an array.

For example:
```php
if (is_wp_error($response)) {
    $error_message = $response->get_error_message();
    $this->log->log("WP-Error in the request: $error_message");
} elseif ($response['response']['code'] === 200) {
    $this->log->log("The update on save request has been processed correctly");
    return TRUE;
} else {
    $this->log->log("Error in the request: " . print_r($response, true));
}
```

Reply:
I have no problem changing it, but I have tested it on a customer and we have not had that problem. It gave him an error and he logged in fine.

Reply:
I agree with @pedromcp90 on this one, I think it's better to check `is_wp_error` first :smile: